### ISSUE TITLE
Image magic converter is failing

### ISSUE THREAD

**Issue by jmaferreira** [keeps/roda#899]
```
Command [/usr/bin/convert, -regard-warnings, /root/.roda/data/storage/aip/400d630e-dc12-47b0-ad4b-a48635eff978/representations/rep1/data/Screen, Shot, 2017-04-20, at, 10.28.46.png, tif:/usr/local/tomcat/temp/converted4307374196421432859.tif] terminated with error code 1
Command [/usr/bin/convert, -regard-warnings, /root/.roda/data/storage/aip/400d630e-dc12-47b0-ad4b-a48635eff978/representations/rep1/data/Screen, Shot, 2017-04-20, at, 15.17.54.png, tif:/usr/local/tomcat/temp/converted5537164926052395001.tif] terminated with error code 1
```

Corpora attached.
[Archive.zip](https://github.com/keeps/roda/files/944028/Archive.zip)

**Comment by jmaferreira** [comment]
It appears to be the filename not being escaped.

**Comment by hsilva-keep** [comment]
@nunovieira220 white spaces???

**Comment by jmaferreira** [comment]
Introduced bug?

**Comment by hsilva-keep** [comment]
Not tested properly?

**Comment by nunovieira220** [comment]
Solved on 11b18f91fb9f6de31b5e8cab860642fc67b66e69. Can you test it @jmaferreira ?

**Comment by hsilva-keep** [comment]
Tested. It works.
